recipes/goodtables/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "goodtables" %}
+{% set version = "2.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e363d0d2762f5ac48b5f408bfe45c0de1ce8bad908cff42464bb11616ad40e55
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - goodtables = goodtables.cli:cli
+  script: "{{ PYTHON }} -m pip install . -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - six >=1.9
+    - click >=6.6
+    - click-default-group
+    - requests >=2.10
+    - simpleeval >=0.9
+    - statistics >=1.0
+    - tabulator >=1.20
+    - tableschema >=1.0.3
+    - datapackage >=1.2
```

Comment:
Trying to debug this on my Windows machine I was not able to install an env with these dependencies b/c of:

```yaml
  - statistics >=1.0
  - tabulator >=1.20
  - tableschema >=1.0.3
  - datapackage >=1.2
```

I'm checking the package now to see what is going on...

Reply:
The first one, statistic, is a python 2 only package!

Reply:
Looks like many of those depend on `openpyxl` and that one may need to be rebuilt. Working on it...